config/prow/jobs/custom/test-infra.yaml

```diff
@@ -458,7 +479,39 @@ periodics:
       secret:
         secretName: prow-updater-robot-ssh-key
         defaultMode: 0400
-
+- cron: "0 */6 * * *" # Every 6 hours
+  name: ci-knativeteam-groups
+  cluster: prow-trusted
+  decorate: true
+  max_concurrency: 1
+  extra_refs:
+  - org: knative
+    repo: community
+    base_ref: main
+  annotations:
+      testgrid-dashboards: utilities
+      testgrid-tab-name: knativeteam-groups-jobs
+      testgrid-alert-email: "serverless-engprod-sea@google.com"
+  # rerun_auth_config:
+  #   github_team_slugs:
+  #   # proxy for sig-k8s-infra-oncall
+  #   - org: kubernetes
+  #     slug: sig-k8s-infra-leads
+  #   # team specifically for this service
+  #   - org: kubernetes
+  #     slug: k8s-infra-group-admins
+  spec:
+    serviceAccountName: gsuite-groups-manager
```

Comment:
Do you have a GCP SA in mind to associate with this k8s SA for workload identity? Or do you need me to create one?

Reply:
yes, create a gsuite-groups-manager and bind it.

Reply:
I have created both the k8s SA and GSA, and configured binding for them:

```
chizhg@chizhg-macbookpro:~$ gcloud iam service-accounts add-iam-policy-binding gsuite-groups-manager@knative-tests.iam.gserviceaccount.com \
    --role roles/iam.workloadIdentityUser \
    --member "serviceAccount:knative-tests.svc.id.goog[test-pods/gsuite-groups-manager]" --project=knative-tests
Updated IAM policy for serviceAccount [gsuite-groups-manager@knative-tests.iam.gserviceaccount.com].
bindings:
- members:
  - serviceAccount:knative-tests.svc.id.goog[test-pods/gsuite-groups-manager]
  role: roles/iam.workloadIdentityUser
etag: BwXXdVA4qRI=
version: 1
chizhg@chizhg-macbookpro:~$ kubectl annotate serviceaccount gsuite-groups-manager \
    --namespace test-pods \
    iam.gke.io/gcp-service-account=gsuite-groups-manager@knative-tests.iam.gserviceaccount.com
serviceaccount/gsuite-groups-manager annotated
```

Reply:
Nice, we can go ahead and merge this.

Reply:
The periodic Prow job will start failing for sure, but it's okay to me since it won't impact anything.